custom_components/alexa_media/__init__.py

```
@@ -3231,11 +3232,13 @@ async def test_login_status(hass, config_entry, login) -> bool:
         elaspsed_time: str = str(datetime.now() - login.stats.get("login_timestamp"))
         api_calls: int = login.stats.get("api_calls")
         message += f"Relogin required after {elaspsed_time} and {api_calls} api calls."
+    host = urlparse(login.url).hostname or login.url
+    notification_id=f"alexa_media_{slugify(login.email)}_{slugify(host)}"
     async_create_persistent_notification(
         hass,
         title="Alexa Media Reauthentication Required",
         message=message,
-        notification_id=f"alexa_media_{slugify(login.email)}{slugify(login.url[7:])}",
+        notification_id,
     )
```

Pylint (4.0.5)

[error] Parsing failed: 'positional argument follows keyword argument (alexa_media.__init__, line 3242)'

(E0001)